server/socket/listeners/user.js

```diff
@@ -35,9 +35,12 @@ const userListeners = (socket) => {
 		logger.info('Password Reset Listener running...');
 		const response = await passwordReset(user);
 		if (response.error) {
-			socket.emit('notify_error', response);
+			response.error.forEach((error) => {
+				socket.emit('notificationError', error);
+			});
 		} else {
-			socket.emit('passwordReset', response);
+			socket.emit('passwordReset', {});
+			socket.emit('notification_success', 'Password reset successful, you can now login !');
```

Comment:
'notification_success' has no listener counterpart on the client. Do you mean 'notificationSuccess' ?